### DISCUSSION TITLE
Different compile size result between 2.0.3 and 2.0.4

### DISCUSSION THREAD

**Discussion by buffcode** [espressif/arduino-esp32#6952] [Q&A]
Hi,

I am quite new to Arduino development and am using a `doitESP32devkitV1`.

When updating from board version 2.0.3 to 2.0.4, my compile size changed from 97% to 108%. Other way round, I can fix it by downgrading to 2.0.3.

Is this expected? Can I use the new board version and disable some flags/... do still be in range of the board size?

Thanks in advance!

**Comment by me-no-dev** [comment]
This is not expected. Could you please elaborate a bit what your sketch does and uses. Maybe we can find the part that is causing it

**Reply by buffcode** [reply]
It is a simple BLE scanner that transmits the found devices via HTTP and WiFi to another client.

The sketch supports AVR and ESP32 boards, the project is structured into multiple files (classes, helper functions, …).

If there is anything else I can provide please let me know.
